packages/cli/index.js

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+'use strict'
+
+const cosmiconfig = require('cosmiconfig')
+const debug = require('debug')('@tracespace/cli')
+const cli = require('./lib/cli')
+
+debug('Searching for default config')
+
+cosmiconfig('tracespace')
+  .search()
+  .then(rc => {
+    debug('Default config', rc)
+    return rc ? Object.assign({_configFile: rc.filepath}, rc.config) : {}
```

Comment:
I guess you are comfortable with moving to ES6 because I am seeing `Object.assign` in quite a few places.

Reply:
Since this is expressly a Node package, I'm comfortable with any ES6 that is supported by Node v6 LTS.

I like that it was `Object.assign` that you noticed and not all the `const`s and arrow functions hah